src/component/admin/users/UsersList/UsersList.tsx

```diff
@@ -116,136 +97,233 @@ const UsersList = ({ search }: IUsersListProps) => {
+                width: '40%',
+                Cell: HighlightCell,
+            },
+            {
+                id: 'username',
+                Header: 'Username',
+                accessor: (row: any) => row.username || row.email,
+                width: '40%',
+                Cell: HighlightCell,
+            },
+            {
+                id: 'role',
+                Header: 'Role',
+                accessor: (row: any) =>
+                    roles.find((role: IRole) => role.id === row.rootRole)
+                        ?.name || '',
+                disableGlobalFilter: true,
+            },
+            {
+                id: 'last-login',
+                Header: 'Last login',
+                accessor: (row: any) => row.seenAt || '',
+                Cell: ({ row: { original: user } }: any) => (
+                    <TimeAgoCell value={user.seenAt} emptyText="Never logged" />
+                ),
+                sortType: 'date',
+                disableGlobalFilter: true,
+            },
+            {
+                Header: 'Actions',
+                id: 'Actions',
+                align: 'center',
+                Cell: ({ row: { original: user } }: any) => (
+                    <Box sx={{ display: 'flex', justifyContent: 'center' }}>
+                        <PermissionIconButton
+                            data-loading
+                            onClick={() => {
+                                navigate(`/admin/users/${user.id}/edit`);
+                            }}
+                            permission={ADMIN}
+                            tooltipProps={{
+                                title: 'Edit user',
+                            }}
+                        >
+                            <Edit />
+                        </PermissionIconButton>
+                        <PermissionIconButton
+                            data-loading
+                            onClick={openPwDialog(user)}
+                            permission={ADMIN}
+                            tooltipProps={{
+                                title: 'Change password',
+                            }}
+                        >
+                            <Lock />
+                        </PermissionIconButton>
+                        <PermissionIconButton
+                            data-loading
+                            onClick={openDelDialog(user)}
+                            permission={ADMIN}
+                            tooltipProps={{
+                                title: 'Remove user',
+                            }}
+                        >
+                            <Delete />
+                        </PermissionIconButton>
+                    </Box>
+                ),
+                width: 100,
+                disableSortBy: true,
+            },
+        ],
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+        [loading, navigate, isBillingUsers]
```

Comment:
Was having some trouble with these dependencies. For example, `roles` was suggested as a dependency but when adding it I got an infinite (render?) loop React error. 
These current dependencies make logical sense to me, but maybe someone more experienced with React has a better solution here.